ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3327,6 +3327,23 @@ HWRWC:
   implements:
   - CONTROL
 
+PHWVVSC:
+  guid: "b1fecf41-bedf-48e6-9e8c-ab0705c4c6d2"
+  description: "Preheating water valve voltage monitoring on supply air side."
+  is_abstract: true
+  opt_uses:
+  - heating_request_count
+  - leaving_air_preheating_coil_temperature_sensor
+  - high_supply_air_temperature_alarm
+  - low_supply_air_temperature_alarm
+  - preheating_run_command
+  uses:
+  - preheating_water_valve_voltage_command
```

Comment:
is there a sensor?

Reply:
No, we have not seen a sensor in the building data and same applies to other abstracts. 
Is a sensor required to appropriately model this abstract? If so, does adding a percentage_sensor make sense?

Reply:
Since we have the temperature control fields, its fine for this one. Just wanted to confirm.